jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -55,11 +55,11 @@ blockchain_configuration:
   # total fees: constant + (num_inputs + num_outputs) * coefficient [+ certificate]
   linear_fees:
     # this is the minimum value to pay for every transaction
-    constant: 2
+    constant: 200000
     # the additional fee to pay for every inputs and outputs
-    coefficient: 1
+    coefficient: 100000
```

Comment:
I don't understand why you needed to change this value?

Reply:
See above.  This is 0.1 Ada.

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.